monero-node/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      # <app-id>_<web-container-name>_1
+      # e.g. 'btc-rpc-explorer_web_1'
+      # Note that the '_1' at the end is needed
+      APP_HOST: monero-node_monerod_1
+      APP_PORT: 18089
+      PROXY_AUTH_ADD: "false"
+  
+  monerod:
+    image: sethsimmons/simple-monerod:v0.18.2.2
+    command: "--rpc-restricted-bind-ip=0.0.0.0 --rpc-restricted-bind-port=18089 --public-node --no-igd --no-zmq --enable-dns-blocklist"
```

Comment:
Due to the existing storage requirements for running a Bitcoin node, would probably be ideal to default to a pruned node for this implementation.

Simply add `--prune-blockchain` to the end of this line to enable it.

There are no real downsides for the user to run a pruned node:

https://www.getmonero.org/resources/moneropedia/pruning.html

Reply:
you're right, I just changed it!